platform-common/src/main/java/com/softicar/platform/common/core/exception/ExceptionsCollector.java

```diff
@@ -98,4 +105,31 @@ public void consumeIfNotEmpty(Consumer<String> messageConsumer) {
 			messageConsumer.accept(getMessage());
 		}
 	}
+
+	public static class ExceptionCollection {
```

Comment:
This `public static class` could be extracted to a separate file.

Reply:
ok, I completely redesigned the code an removed the collection class ;)

Reply:
nice